website/react-magma-docs/src/pages/api/modal.mdx

```diff
@@ -343,24 +364,24 @@ export function Example() {
         isOpen={showModal}
         isInverse
       >
-        <p>This is a modal, doing modal things.</p>
-        <p>
+        <Paragraph noMargins>This is a modal, doing modal things.</Paragraph>
```

Comment:
We need to add `isInverse` to all these Paragraphs
![image](https://github.com/cengage/react-magma/assets/91160746/6b43effc-7892-4ecc-9b2c-1632d18b16e8)